bmp280/78afeb33-03cb-4d10-9457-16666f03451b.cpp

```diff
@@ -0,0 +1,5 @@
+float getAltitude{{ instance }}() {
+    const float seaLevelhPa = 1013.25; // Adjusted to your sea level pressure, may need to be changed
```

Comment:
I think it might be best to set this as a global constant in `instantiate.cpp`, then reference it in this function.

*`instantiate.cpp`:*

```cpp
//TODO: Update to current sea level pressure
static const float SEA_LEVEL_hPa = 1013.25;

Adafruit_BMP280 bmp{{ instance }}; // I2C
```